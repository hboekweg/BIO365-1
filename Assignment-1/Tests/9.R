# This imports your function
source("9.R")

# This invokes your function
print(shouldIEat(9.5))
print(shouldIEat(8.0))
print(shouldIEat(7.9))
print(shouldIEat(3.0))
print(shouldIEat(1.1))
print(shouldIEat(11))
print(shouldIEat(-2))
