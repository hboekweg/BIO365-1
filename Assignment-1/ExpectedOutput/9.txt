[1] "Go eat!"
[1] "Go eat!"
[1] "Don't eat!"
[1] "Don't eat!"
[1] "Take a nap!"
[1] "Invalid input."
[1] "Invalid input."
